### PR TITLE
added signup page

### PR DESCRIPTION
**Pull request by tanikka-robinson**
UI is liked this
image

### CODE REVIEW

File: src/components/button/index.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.custom-button {
+  display: flex;
+  width: 100%;
+  height: 70px;
```
Comment (edjose-chen) [inline src/components/button/index.scss:4]: we can use rem instead of constant px.

File: src/pages/login/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { TabForm } from "../../components/tabs";
+import { InputForm } from "../../components/input";
+import { Button } from "../../components/button";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FaCheck } from "react-icons/fa";
+import logoSignup from "../../assets/signup-vector-top.svg";
+export const LoginField = () => {
+  const [checked, setChecked] = useState(true);
+  console.log(checked);
```
Comment (edjose-chen) [inline src/pages/login/index.js]: remove this console

File: src/pages/login/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <div className="login-container">
+      <div className="logo-field">
+        <img src={logo} alt="RBR"></img>
```
Comment (edjose-chen) [inline src/pages/login/index.js]: we can simply end /> rather than use </img>

File: src/pages/login/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      </div>
+      <div className="login-field-container">
+        <img src={logoSignup} alt="vector" className="logo-signup"></img>
+        <div className="login-field-empty"></div>
+        <div className="login-field">
+          <div className="login-field-header">
+            <div className="open-label">Log In</div>
+            <div className="started-label">Please login to continue</div>
+            <div className="login-field-header-tab-container"></div>
```
Comment (edjose-chen) [inline src/pages/login/index.js]: same here. just add /> instead </div> when no child

File: src/pages/login/index.scss [hunk 5]
```diff
@@ -0,0 +1,173 @@
+.login-container {
+  width: 1440px;
```
Comment (edjose-chen) [inline src/pages/login/index.scss]: if device width<1440px then it would be overlap

File: src/pages/login/index.scss [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+      }
+    }
+    .login-info-container {
+      display: flex;
+      flex: 1;
+    }
+  }
+}
+
+// .login-container .logo-field {
```
Comment (edjose-chen) [inline src/pages/login/index.scss]: remove if this comment is not kinda code description

File: src/pages/recovery/index.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+import { TabForm } from "../../components/tabs";
+import { InputForm } from "../../components/input";
+import { Button } from "../../components/button";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FaCheck } from "react-icons/fa";
+import logoSignup from "../../assets/signup-vector-top.svg";
+export const RecoveryField = () => {
+  const [checked, setChecked] = useState(true);
+  console.log(checked);
```
Comment (edjose-chen) [inline src/pages/recovery/index.js]: remove log

File: src/pages/recovery/index.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+  return (
+    <div className="recovery-container">
+      <div className="logo-field">
+        <img src={logo} alt="RBR"></img>
```
Comment (edjose-chen) [inline src/pages/recovery/index.js]: we should use custom component to keep DRY and KISS principle

File: src/App.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+import { SignupField } from "./pages/signup";
+import { RecoveryField } from "./pages/recovery";
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/" element={<Splash />}></Route>
+        <Route path="/login" element={<LoginField />}></Route>
+        <Route path="/signup" element={<SignupField />}></Route>
+        <Route path="/recovery" element={<RecoveryField />}></Route>
```
Comment (terex-x) [inline src/App.js]: <Route exact path

File: src/components/input/index.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import "./index.scss";
+export const InputForm = ({ label, type }) => {
+  const [ispassword, setIsPassword] = useState(true);
+  const [toggle, setToggle] = useState(true);
```
Comment (terex-x) [inline src/components/input/index.js]: rename this to `InputGroup`/`InputField`

Comment (terex-x) [inline src/components/input/index.js]: ispassword -> isPassword

File: src/pages/login/index.js (hunk 2 above)
Comment (SnowindPassion) [reply src/pages/login/index.js]: import { TabFrom, InputForm, Button } from "../../components"

**Comment by tanikka-robinson** [comment]
This ui is liked this.
image

**Pull request merged**